src/scripts/apply.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Check CLI config file
+if [[ -n "${TF_PARAM_CLI_CONFIG_FILE}" ]]; then
+    if [[ -f "${TF_PARAM_CLI_CONFIG_FILE}" ]]; then
+        export TF_CLI_CONFIG_FILE=${TF_PARAM_CLI_CONFIG_FILE}
+    else
+        echo "Terraform cli config does not exist: ${TF_PARAM_CLI_CONFIG_FILE}"
+        exit 1
+    fi
+fi
+# 'path' is a required parameter, save it as module_path
+readonly module_path="${TF_PARAM_PATH}"
+export path=$module_path
+
+if [[ ! -d "$module_path" ]]; then
+    echo "Path does not exist: $module_path"
+    exit 1
+fi
+# the following is needed to process backend configs
+if [[ -n "${TF_PARAM_BACKEND_CONFIG_FILE}" ]]; then
+    for file in $(echo "${TF_PARAM_BACKEND_CONFIG_FILE}" | tr ',' '\n'); do
+        if [[ -f "$file" ]]; then
+            INIT_ARGS="$INIT_ARGS -backend-config=$file"
+        elif [[ -f "$module_path/$file" ]]; then
+            INIT_ARGS="$INIT_ARGS -backend-config=$module_path/$file"
```

Comment:
also, be careful, https://github.com/CircleCI-Public/terraform-orb/pull/46/files/0741edfd0a76892aff9462dd9666656d2e1de639#r677491870 also means that you no longer need to add the $module_path prefix in path you give to the commands. Here this will make Terraform searach in ${module_path}/${module_path} as the first comes from chdir itself.